cluster-autoscaler/cloudprovider/azure/azure_cloud_provider_test.go

```diff
@@ -55,18 +57,19 @@ func newTestAzureManager(t *testing.T) *AzureManager {
 		env:                  azure.PublicCloud,
 		explicitlyConfigured: make(map[string]bool),
 		config: &Config{
-			ResourceGroup:       "rg",
-			VMType:              vmTypeVMSS,
+			Config: providerazure.Config{
+				ResourceGroup: "rg",
+				VMType:        providerazureconsts.VMTypeVMSS,
+			},
 			MaxDeploymentsCount: 2,
 			Deployment:          "deployment",
-			EnableForceDelete:   true,
-			Location:            "eastus",
+			EnableForceDelete:   false,
```

Comment:
Why change?

Reply:
A mistake in merge conflict resolving it seems. Fixed along others.